packages/host/app/components/operator-mode/container.gts

```diff
@@ -143,12 +143,24 @@ export default class OperatorModeContainer extends Component<Signature> {
       @boxelModalOverlayColor='var(--operator-mode-bg-color)'
     >
       <CardCatalogModal />
-      {{#if (and this.matrixService.isLoggedIn this.isCodeMode)}}
+      {{#if
+        (and
+          this.matrixService.isLoggedIn
+          (not this.operatorModeStateService.needsAuthorization)
```

Comment:
What is the difference between this `operatorModeStateService.needsAuthorization` and `matrixService.isLoggedIn`? Is it referring to realm authorization? If so, can we express this in a combined, more semantic way rather than referring to the two bools in the template?

Reply:
yes it does refer to the realm authorization which is not necessarily the same as the matrix server authentication. yeah I can combine this before it hits the template

Reply:
(a good example is a public realm--you don't have to be logged in in order to have realm authorization)

Reply:
I came up with a better name for this and added a comment around why we need this  in https://github.com/cardstack/boxel/pull/1615/commits/7e927be1b4f2d54d415731685bf3a5b0db2f284c